packages/block-library/src/breadcrumbs/edit.js

```diff
@@ -155,12 +176,7 @@ export default function BreadcrumbEdit( {
 			placeholderItems.push( __( 'Ancestor' ), __( 'Parent' ) );
 		}
 		placeholder = (
-			<nav
-				style={ {
-					'--separator': `'${ separator }'`,
-				} }
-				inert="true"
-			>
+			<nav { ...blockProps } inert="true">
```

Comment:
```suggestion
			<nav { ...blockProps }>
```

The inert attribute must be removed because it disables interaction with the block. The `a` element inside the block has its interactions disabled by the `useDisabled` hook, so it is safe to remove the `inert` attribute.

Reply:
Alright! But `inert` is used for placeholder block not for HTML rendered content, I see it does not have any such links though, so as per your suggestion we can remove that.

Thank You,